src/tsickle.ts

```diff
@@ -2017,6 +2025,41 @@ export function emitWithTsickle(
   };
 }
 
+function addClutzAliases(
+    dtsFileContent: string, sourceFiles: ts.SourceFile[], typeChecker: ts.TypeChecker,
+    host: TsickleHost): string {
+  let reexports = '';
+  for (const sf of sourceFiles!) {
```

Comment:
Why do you need ! in `sourceFiles!`. It look non-nullable to me `ts.SourceFile[]`